lib/presentation/settings/settings_screen.dart

```diff
@@ -1,16 +1,16 @@
 import 'package:auto_route/auto_route.dart';
-import 'package:collaction_app/application/user/profile/profile_bloc.dart';
-import 'package:collaction_app/infrastructure/core/injection.dart';
-import 'package:collaction_app/presentation/settings/widgets/share_collaction_list_tile.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 import '../../application/auth/auth_bloc.dart';
+import '../../application/user/profile/profile_bloc.dart';
+import '../../infrastructure/core/injection.dart';
 import '../core/collaction_icons.dart';
 import '../routes/app_routes.gr.dart';
 import '../shared_widgets/custom_app_bars/custom_appbar.dart';
 import '../themes/constants.dart';
+import 'widgets/share_collaction_list_tile.dart';
 
 const shareText =
     "Check out CollAction at https://play.google.com/store/apps/details?id=org.collaction.collaction_app for Android and https://apps.apple.com/app/id1597643827 for iOS. Let's solve all collective action problems in the world.";
```

Comment:
`shareText` and `shareEmailSubject` are not used and should be removed.

Reply:
Ok

Reply:
I'm not sure it will work adding it to CI Pipeline, i had to do some  manual formatting of `oss_licenses.dart` before CI passed

Reply:
**Documentation:**  
Mention in readme to generated OSS licenses with `flutter pub run flutter_oss_licenses:generate.dart` (see https://github.com/CollActionteam/collaction_app/blob/development/docs/README.md#code-generation)

**CI workflow:**
change
```
- name: Build generated code
   run: flutter pub run build_runner build --delete-conflicting-outputs
- name: Check for any formatting issues
   run: flutter format --dry-run --set-exit-if-changed .
```
to
```
- name: Check for any formatting issues
   run: flutter format --dry-run --set-exit-if-changed .
- name: Build generated code
   run:   |
     flutter pub run flutter_oss_licenses:generate.dart
     flutter pub run build_runner build --delete-conflicting-outputs
```
(see https://github.com/CollActionteam/collaction_app/blob/development/.github/workflows/ci.yml#L43)